lhotse/workflows/forced_alignment/mms_aligner.py

```diff
@@ -0,0 +1,206 @@
+import logging
+import re
+from typing import List, Optional, Tuple
+
+import torch
+import torchaudio
+
+from lhotse.supervision import AlignmentItem
+from lhotse.utils import is_module_available
+
+from .base import FailedToAlign, ForcedAligner
+
+# Note: Korean _does_ use spaces, but not so straightforwardly as Indo-European languages.
+# We still need to tokenize it into morphemes to get proper alignment.
+LANGUAGES_WITHOUT_SPACES = ["zh", "ja", "ko", "th", "my", "km", "lo"]
+
+
+class MMSForcedAligner(ForcedAligner):
+    def __init__(self, bundle_name: str, device: str = "cpu"):
```

Comment:
I'd remove `bundle_name` from param list and hardcode `self.bundle_name = "MMS_FSA"`, this will allso allow to remove the assertion below.

Reply:
I'd consider option `check_language: bool = True` which would warn the users about missing `language` field in the supervisions if detected (the message should mention how to disable the warnings as well).